### PR TITLE
github action to copy readme files to arc-docs

### PR DESCRIPTION
**Pull request by saurabhjindal-SF**

### CODE REVIEW

File: .github/workflows/update-docs.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          destination-repository-name: 'terraform-aws-ref-arch-network'
+          target-directory: 'docs/arc-iac-docs/modules/terraform-aws-ref-arch-network/example'
+          user-email: saurabh.jindal@gmail.com
+          user-name: 'saurabhjindal-SF'
+          target-branch: main
+          commit-message: ${{ github.event.head_commit.message }}
+      - name: Pushes to another repository
+        uses: cpina/github-action-push-to-another-repository@main
+        env:
+          API_TOKEN_GITHUB: ${{ secrets.ARC_DOCS_TOKEN }}
```
Comment (mayank0202) [inline .github/workflows/update-docs.yml:39]: `GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}`

File: .github/workflows/update-docs.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        uses: actions/checkout@v3
+      - name: Pushes Readme file
+        uses: dmnemec/copy_file_to_another_repo_action@main
+        env:
+          API_TOKEN_GITHUB: ${{ secrets.ARC_DOCS_TOKEN }}
+        with:
+          source_file: 'README.md'
+          destination_repo: 'sourcefuse/arc-docs'
+          destination_folder: 'docs/arc-iac-docs/modules/terraform-aws-ref-arch-network'
+          user_email: 'saurabh.jindal@gmail.com'
```
Comment (mayank0202) [inline .github/workflows/update-docs.yml:20]: Please modify this to use the PR author's info.

File: .github/workflows/update-docs.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      - name: Pushes example folder
+        uses: cpina/github-action-push-to-another-repository@main
+        env:
+          API_TOKEN_GITHUB: ${{ secrets.ARC_DOCS_TOKEN }}
+        with:
+          source-directory: 'example'
+          destination-github-username: 'sourcefuse'
+          destination-repository-name: 'terraform-aws-ref-arch-network'
+          target-directory: 'docs/arc-iac-docs/modules/terraform-aws-ref-arch-network/example'
+          user-email: saurabh.jindal@gmail.com
```
Comment (mayank0202) [inline .github/workflows/update-docs.yml:32]: Please modify this to use the PR author's info.

File: .github/workflows/update-docs.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      - name: Pushes to another repository
+        uses: cpina/github-action-push-to-another-repository@main
+        env:
+          API_TOKEN_GITHUB: ${{ secrets.ARC_DOCS_TOKEN }}
+        with:
+          source-directory: 'static'
+          destination-github-username: 'sourcefuse'
+          destination-repository-name: 'terraform-aws-ref-arch-network'
+          target-directory: 'docs/arc-iac-docs/modules/terraform-aws-ref-arch-network/static'
+          user-email: saurabh.jindal@gmail.com
```
Comment (mayank0202) [inline .github/workflows/update-docs.yml:45]: Please modify this to use the PR author's info.

File: .version [hunk 5]
```diff
@@ -1 +1 @@
-2.1.5
+2.1.6
```
Comment (mayank0202) [inline .version:1]: give `2.1.7`

**Review by mayank0202 — CHANGES_REQUESTED** [review]

**Review by mayank0202 — APPROVED** [review]

**Review by jamescrowley321 — APPROVED** [review]

**Pull request merged**